cpp/arcticdb/column_store/column.hpp

```diff
@@ -357,6 +357,17 @@ class Column {
         std::swap(data_.buffer(), new_buffer);
     }
 
+    inline void set_sparse_block(ChunkedBuffer&& buffer, util::BitSet&& bitset) {
+        data_.buffer() = std::move(buffer);
```

Comment:
Why the std::move? `buffer` and `bitset` are already rvalues, no?

Reply:
https://stackoverflow.com/questions/56316766/why-is-a-named-rvalue-reference-an-lvalue-expression